components/brave_wallet_ui/components/shared/nft-icon/nft-icon.tsx

```diff
@@ -54,7 +55,11 @@ export const NftIcon = (props: NftIconProps) => {
   const nftIframeUrl = React.useMemo(() => {
     const urlParams = new URLSearchParams({
       displayMode: 'icon',
-      icon: encodeURI(remoteImage || '')
+      // appears to still be broken. Seems this is being used in brave://wallet/crypto/accounts/<account-id>/nfts to display nft images
```

Comment:
can you elaborate on this comment? it was broken before and still broken with this PR?

Reply:
That was added by me. Based on the images that were shared in the DM, I don't think it's needed anymore.

Reply:
Was a leftover comment from @kdenhartog, removed it!